packages/perseus-score/package.json

```diff
@@ -27,7 +27,9 @@
     "dependencies": {
         "@khanacademy/kas": "workspace:*",
         "@khanacademy/kmath": "workspace:*",
-        "@khanacademy/perseus-core": "workspace:*"
+        "@khanacademy/perseus-core": "workspace:*",
+        "@khanacademy/perseus-utils": "workspace:*",
+        "perseus-build-settings": "workspace:*"
```

Comment:
I noticed that sometimes this is a dev dep and here it's a regular dep. Is one preferred over the other?

Reply:
`purseus-build-settings` should be a dev dep!  I'll review and fix where it's wrong.

Reply:
I also caught one of `perseus-utils` in the wrong spot and fixed that too!